reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -124,11 +178,53 @@ class StaticHtmlReporter : Reporter {
                         issueTable(reportTableModel.issueSummary)
                     }
 
+                    h2 {
+                        id = "project-dependencies"
+                        +"Projects and Dependencies"
+                    }
+
+                    details {
+                        unsafe {
+                            +"<summary>Project and Dependency Index</summary>"
+                        }
+                        ul {
+                            reportTableModel.projectDependencies.forEach { (project, projectTable) ->
+                                li {
+                                    a("#${project.id.toCoordinates()}") {
+                                        +project.id.toCoordinates()
+
+                                        if (projectTable.isExcluded()) {
+                                            projectTable.pathExcludes.forEach { exclude ->
+                                                +" "
+                                                div("ort-reason") { +"Excluded: ${exclude.reason} - ${exclude.comment}" }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+
                     reportTableModel.projectDependencies.forEach { (project, table) ->
                         projectTable(project, table)
                     }
 
+                    if (reportTableModel.metadata.isNotEmpty()) {
+                        metadataTable(reportTableModel.metadata)
+                    }
+
                     repositoryConfiguration(reportTableModel.config)
+
+                    div {
```

Comment:
Right above the code calls functions like `projectTable` and `metadataTable`. In order to keep the abstraction level consistent please also move the code for that `footer` block into a separate function.

Reply:
Done